Node.JS installattion instructions

dbkeys opened the pull request:
Expand README.md with Node.JS installation instructions for Debian / Ubuntu.

Put rpcAuth.js in the .gitignore file so passwords don't get uploaded to github.

Commit:
More detailed Node.JS / installation instructions

Commit:
More detail on Debian / Ubuntu Node.JS install

Commit:
Formatting

Commit:
More formatting

Commit:
Learning Formatting

Commit:
Triple Back Tick magic

sneurlax commented:
rpcAuth only contains sample auth credentials

use this command to ignore updates (ie., your credentials)
`git update-index --assume-unchanged rpcAuth.js`

sneurlax committed:
Add sub-headings and mark codeblocks as bash

sneurlax committed:
unignore rpcAuth.js

rpcAuth only contains sample auth credentials

use this command to ignore updates (ie., your credentials)
`git update-index --assume-unchanged rpcAuth.js`

sneurlax committed:
mention assume-unchanged
